src/js/BookReader/Mode2Up.js

```diff
@@ -153,59 +205,27 @@ export class Mode2Up {
     this.br.refs.$brContainer.dragscrollable({preventDefault:true});
     this.br.bindGestures(this.br.refs.$brContainer);
 
-    // $$$ calculate first then set
-    this.br.refs.$brTwoPageView.css({
-      height: `${this.br.twoPage.totalHeight}px`,
-      width: `${this.br.twoPage.totalWidth}px`,
-      position: 'absolute'
-    });
+    const { mainContainerCSs, spreadCoverCss, leafEdgeRCss, leafEdgeLCss, spineCss } = this.spreadCSSAttributes();
 
-    // If there will not be scrollbars (e.g. when zooming out) we center the book
-    // since otherwise the book will be stuck off-center
-    if (this.br.twoPage.totalWidth < this.br.refs.$brContainer.prop('clientWidth')) {
```

Comment:
i removed this check and added default values to the incoming args...
seems like the only call that adds X & Y coords is when zoom calls it.